File: Java_Advanced/gsapi/src/main/java/br/com/fiap/gs/gsapi/dto/stats/CategoryCountDTO.java
// Arquivo: src/main/java/br/com/fiap/gs/gsapi/dto/stats/CategoryCountDTO.java
package br.com.fiap.gs.gsapi.dto.stats;

public class CategoryCountDTO {
    private String categoryTitle;
    private long count;

    public CategoryCountDTO() {
    }

    public CategoryCountDTO(String categoryTitle, long count) {
        this.categoryTitle = categoryTitle;
        this.count = count;
    }

    // Getters e Setters
    public String getCategoryTitle() {
        return categoryTitle;
    }

    public void setCategoryTitle(String categoryTitle) {
        this.categoryTitle = categoryTitle;
    }

    public long getCount() {
        return count;
    }

    public void setCount(long count) {
        this.count = count;
    }
}

File: Advanced_Business_Development_with.NET/gsApi/gsApi/dto/response/CategoryCountDto.cs
// File: SeuProjetoNET/DTOs/Response/CategoryCountDto.cs
namespace gsApi.model.DTOs.Response
{
    public class CategoryCountDto
    {
        public string CategoryTitle { get; set; }
        public long Count { get; set; }

        public CategoryCountDto(string categoryTitle, long count)
        {
            CategoryTitle = categoryTitle;
            Count = count;
        }
    }
}